frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -209,7 +211,7 @@ const Sidebar = ({
         { accounts.length ? (
           <div key="buy" className={style.sidebarItem}>
             <NavLink
-              className={({ isActive }) => isActive ? style.sidebarActive : ''}
+              className={({ isActive }) => isActive || userInSpecificAccountBuyPage ? style.sidebarActive : ''}
```

Comment:
At this point we don't actually need the `isActive` here, as `userInSpecificAccountBuyPage` checks for all urls prefixed by `/buy`. 

Keeping for consistency,  but lmk if we wanna remove it.

Reply:
Not sure which is best, actually. It's ok to leave it, I guess.